components/image-gallery.js

```diff
@@ -1,29 +1,26 @@
 import { Image } from 'react-datocms'
 import { Parallax } from 'react-scroll-parallax';
 
-
 export default function ImageGallery({ images }) {
   let index = 0;
 
   function addRandomPos(){
     let classN = 'shadow-small hover:shadow-medium transition-shadow duration-200';
     if (index % 2 == 0){
-      classN += ' md:col-start-'+1;
-      classN += ' md:col-end-'+ 4;
+      classN += ' md:col-start-1';
+      classN += ' md:col-end-4';
     } else {
-      classN += ' md:col-start-'+2;
-      classN += ' md:col-end-'+ 5;
+      classN += ' md:col-start-2';
+      classN += ' md:col-end-5';
     }
```

Comment:
checkout https://www.npmjs.com/package/classnames

designed for doing things like this with more of a declarative style.